Sources/SwiftDiagnostics/GroupedDiagnostics.swift

```diff
@@ -172,7 +172,7 @@ extension GroupedDiagnostics {
       tree: sourceFile.tree
     )
 
-    let colorizeBufferOutline = formatter.colorizeBufferOutline
+    let decorateBufferOutline = formatter.diagnosticDecorator.decorateBufferOutline
```

Comment:
I would prefer to call `formatter.diagnosticDecorator.decorateBufferOutline` below multiple times instead of storing the function as a closure. If you do want to abbreviate it, I would be fine with storing the diagnostic decorator in a temporary variable. 

Storing functions in variables just makes it a lot harder to understand what’s going on IMO because you can’t do Quick Help on the call and jump to definition requires two jumps instead of one.

```swift
let decorator = formatter.diagnosticDecorator
…
… = decorator.decorateBufferOutline(…)
```

Reply:
I've added `let diagnosticDecorator = formatter.diagnosticDecorator` instead of storing func inn closure.